java/pom.xml

```diff
@@ -11,8 +11,12 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <arrow.version>3.0.0</arrow.version>
+        <arrow.version>7.0.0-20220223174055-ed836ca-dremio</arrow.version>
```

Comment:
Can we add a jar-with-dependencies configuration for this? There are too many dependencies to add to the classpath for users.